packager/certloader/certloader.go

```diff
@@ -184,9 +174,14 @@ func WriteCertsToFile(certs []*x509.Certificate, filepath string) error {
 		}
 	}()
 
-	cert := certToPEM(certs[0])
-	issuer := certToPEM(certs[1])
-	bundled := append(cert, issuer...)
+	certLen := len(certs)
+	bundled := make([]byte, 0, 0)
+	for i := 0; i < certLen - 1; i++ {
```

Comment:
Can you just change `certLen - 1` to `certLen` and get rid of the special case `issuer` after the loop? Better still use `for _, cert := range certs {`.

Reply:
Done.